lib/pf/constants/syslog.pm

```diff
@@ -107,100 +130,133 @@ our @SyslogInfo = (
             '$syslogtag contains "acct" and $syslogfacility-text == "local2"'
         ]
     },
     {
+        'description' => 'FreeRADIUS CLI server log',
         'name'       => 'radius-cli.log',
         'conditions' =>
           [ '$syslogtag contains "cli" and $syslogfacility-text == "local3"' ]
     },
     {
+        'description' => 'FreeRADIUS eduroam server log',
         'name'       => 'radius-eduroam.log',
         'conditions' => [ '$syslogtag contains "eduroam" ' ]
     },
     {
+        'description' => 'FreeRADIUS load balancing server log (cluster only)',
         'name'       => 'radius-load_balancer.log',
         'conditions' => [
 '$syslogtag contains "load_balancer" and $syslogfacility-text == "local5"'
         ]
     },
     {
+        'description' => 'FreeRADIUS authentication server log',
         'name'       => 'radius.log',
         'conditions' => [
             '$syslogtag contains "auth" and $syslogfacility-text == "local1"',
 '$programname contains "radius" and $syslogfacility-text == "local1"'
         ]
     },
     {
+        'description' => 'Redis global cache logs',
         'name'       => 'redis_cache.log',
         'conditions' => [ '$syslogtag contains "redis-cache"' ]
     },
     {
+        'description' => 'Redis NTLM cache logs',
         'name'       => 'redis_ntlm_cache.log',
         'conditions' => [ '$syslogtag contains "redis-ntlm-cache"' ]
     },
     {
+        'description' => 'Redis queue logs',
         'name'       => 'redis_queue.log',
         'conditions' => [ '$syslogtag contains "redis-queue"' ]
     },
     {
+        'description' => 'Redis server logs',
         'name'       => 'redis_server.log',
         'conditions' => [ '$programname == "redis-server"' ]
     },
     {
+        'description' => 'MariaDB log',
         'name'       => 'mariadb_error.log',
         'conditions' => [ '$syslogtag contains "mysqld"' ],
     },
     {
+        'description' => 'haproxy portal log',
         'name'       => 'haproxy_portal.log',
         'conditions' => [ '$programname == "haproxy" and ($msg contains "portal-http" or $msg contains "backend has no server available")' ],
     },
     {
+        'description' => 'haproxy DB log',
         'name'       => 'haproxy_db.log',
         'conditions' => [ '$programname == "haproxy" and ($msg contains "mysql" or $msg contains "backend has no server available")' ],
     },
     {
+        'description' => 'haproxy admin log',
         'name'       => 'haproxy_admin.log',
         'conditions' => [ '$programname == "haproxy" and ($msg contains "admin-https" or $msg contains "backend has no server available")' ],
     },
 );
 
+our @LOGS = (
+    (map { $_->{name} = "$log_dir/".$_->{name} ; $_ } @SyslogInfo),
+    {
+        'description' => 'Global syslog (CentOS)',
+        'name'       => '/var/log/messages',
+    },
+    {
+        'description' => 'Global syslog (Debian)',
+        'name'       => '/var/log/syslog',
+    },
+);
```

Comment:
It would be nice to have a unique entry with a dynamic `name` attribute that depends on the running OS.